cmd/checkpoint/main.go

```diff
@@ -42,6 +43,8 @@ var (
 )
 
 func main() {
+	flag.Set("logtostderr", "true")
```

Comment:
Yeah. There are also some other issues that logging surfaced. I'll put this in a separate PR